Formula/node.rb

```diff
@@ -100,9 +100,12 @@ def post_install
       cp Dir[libexec/"lib/node_modules/npm/man/#{man}/{npm,package.json,npx}*"], HOMEBREW_PREFIX/"share/man/#{man}"
     end
 
-    npm_root = node_modules/"npm"
-    npmrc = npm_root/"npmrc"
-    npmrc.atomic_write("prefix = #{HOMEBREW_PREFIX}\n")
+    npmrc = <<~EOS
+      prefix = #{HOMEBREW_PREFIX}
+      python = #{Formula["python@2"].opt_bin}/python2\n
```

Comment:
why aren't we just using system python

Reply:
A thx, that was the issue, it worked for me because I have `python@2` installed. But we need to use `python@2` on `MacOS.version <= :snow_leopard`.

Reply:
probably can just set python = python2.7

Reply:
but really I'm fine with python = /usr/bin/python

the `depends_on "python@2" if MacOS.version <= :snow_leopard` at this point just means "this uses python 2"

Reply:
I mean if we always set `python=/usr/bin/python` it will not work on snow leopard or lower.

Reply:
right. that's my point. no one really expects it to.

Reply:
I don't think it's worth trying to support 10.6 and below unless anyone comes complaining, personally. This is one of the few things we actually try to do because it requires minimal effort, but the result of that is likely that _a lot_ of core formulae are 💀on <=10.6 anywhere where the issue is more than extremely minor.